sync-todo/v2/client/kotlin-sdk/app/src/main/res/values/atlasConfig.xml

```diff
@@ -1,4 +1,5 @@
 <resources>
-  <string name="realm_app_id">sync-todo-v2-oxdgx</string>
+  <string name="realm_app_id">****</string>
   <string name="realm_base_url">https://realm.mongodb.com</string>
+  <string name="dataExplorerLink">****</string>
```

Comment:
it looks like the link is configured as appId + baseUrl + atlasUrl in the other SDKs -- should this be similarly constructed? Or a PUT_YOUR_DATA_EXPLORER_LINK_HERE placeholder if it's something users are expected to get?

Reply:
This value is created when the app is created.